### ISSUE TITLE
Continuous training of pre training model

### ISSUE THREAD

**Issue by VampireLLLL** [JiahuiYu/generative_inpainting#387]
Hello Jiahui, thank you for sharing such wonderful works. I read through this paper and have some questions about the structure and implementation. Sorry, I'm new to Deep learning. My question seems a little naive.
I would like to continue training on my dataset with the pre training model you provided. My dataset is now 10000 images. I found that the loss is negative on the loaded pre training model, and the test results will introduce other colors.

Would you like to know if my data set is small or if I need to do wei'tiao.
Please give me some suggestions, thank you very much !!
![04](https://user-images.githubusercontent.com/43810263/71541674-ad267280-2997-11ea-953c-349fdf2d07e9.png)
![4](https://user-images.githubusercontent.com/43810263/71541677-b3b4ea00-2997-11ea-89e2-3634261a4d76.png)

![04](https://user-images.githubusercontent.com/43810263/71541684-c9c2aa80-2997-11ea-98b1-3daf430f1f6e.jpg)

The first picture is the result of the place 2 model test you provided
The second picture is the result of continuous training model test
The last one is the original picture

**Comment by JiahuiYu** [comment]
Hi, yes you can fine tune your own model on your dataset with our pretrained places2 model.

As the result in this image, I guess the failure is that this mask is too large (almost more than 3/5).

**Comment by VampireLLLL** [comment]
Hello Jiahui,I'm sorry to trouble you again！
I'm now fine tuning your place 2 pre training model,I have a few questions I don't understand that need your guidance.
1. I read the parameters of the model, but I don't understand why every name has so many meanings. 
![微信图片_20200103195552](https://user-images.githubusercontent.com/43810263/71722454-c7fb5a00-2e63-11ea-9a8c-99699c1b069b.png)
![微信图片_20200103195557](https://user-images.githubusercontent.com/43810263/71722459-cc277780-2e63-11ea-82f2-4b83049a9fbb.png)
The first picture is loaded with the parameters displayed by the pre training model
The second picture is for me to fine tune the reading parameters
I want to know if I want to freeze some layers, do I need to put all six of them, such as conv7, into freezing? In addition, do I want to freeze all parameter names of the first picture or the second picture
![微信图片_20200103200936](https://user-images.githubusercontent.com/43810263/71722761-fe85a480-2e64-11ea-9a9d-29b28f8beebd.png)
![微信图片_20200103201139](https://user-images.githubusercontent.com/43810263/71722812-47d5f400-2e65-11ea-9a98-dec76bfa46c4.png)
Is it a drawing or a second drawing
2. I'm modifying the model_restorer.py in the neuralgym library. Am I right?

Please forgive me for being a novice. If it's not right, can you tell me how to fine tune it.

3now I want to freeze stage 1 completely. Do you think it will have a good impact on my own data set. Or do you have better suggestions for fine tuning which layers to freeze.
Sorry to bother you again. Thank you for your patience！
